src/title/TitleStream.js

```diff
@@ -68,37 +69,39 @@ const TitleStream = (props: Props) => {
   const _ = useContext(TranslationContext);
 
   return (
-    <View style={componentStyles.outer}>
-      <View style={componentStyles.streamRow}>
-        <StreamIcon
-          style={styles.halfMarginRight}
-          isMuted={!stream.in_home_view}
-          isPrivate={stream.invite_only}
-          color={color}
-          size={20}
-        />
-        <Text style={[styles.navTitle, { color }]} numberOfLines={1} ellipsizeMode="tail">
-          {stream.name}
-        </Text>
-      </View>
-      {isTopicNarrow(narrow) && (
-        <TouchableWithoutFeedback
-          onLongPress={() => {
-            showHeaderActionSheet({
-              showActionSheetWithOptions,
-              callbacks: { dispatch, _ },
-              backgroundData,
-              stream: stream.name,
-              topic: topicOfNarrow(narrow),
-            });
-          }}
-        >
+    <TouchableWithoutFeedback
+      onLongPress={() => {
```

Comment:
We hope to handle long presses when the narrow is a stream narrow too, right? (Whether `showHeaderActionSheet` is made explicitly topic-narrow-only and we grow something similar for stream narrows, or we adapt `showHeaderActionSheet` to be used for both kinds of narrows; I'm looking at discussion at https://github.com/zulip/zulip-mobile/pull/4608/commits/d193f787e9cc58fe9341907b258363aab2935639#r609373770.)

Until that time, it may be worth a small note that we plan to do so, so it doesn't fall off our radar; I think it's nice to avoid no-op press handlers, for how they may interfere with accessibility.

Reply:
> I think it's nice to avoid no-op press handlers, for how they may interfere with accessibility.

Ah, good catch. Yeah, so long as this is a no-op, let's make it simply `undefined` instead (which should have the same effect as not passing it at all): just put the conditional outside the function expression, like `isTopicNarrow(narrow) ? () => { … } : undefined`. That way it won't be there to mislead a11y systems.

Reply:
Done.